sdk/networkanalytics/azure-mgmt-networkanalytics/pyproject.toml

```diff
@@ -4,3 +4,4 @@ mypy = false
 pyright = false
 type_check_samples = false
 verifytypes = false
+ci_enabled = false
```

Comment:
This should be removed until the package has been released again with the deprecation updates. It will need to be added back in post release.

Relevant info:
- https://github.com/Azure/azure-sdk-for-python/blob/main/doc/deprecation_process.md#pyprojecttoml
- https://github.com/Azure/azure-sdk-for-python/blob/main/doc/deprecation_process.md#step-5-create-a-new-pr-to-remove-the-package-from-ci